routes/_pages/settings/general.html

```diff
@@ -24,7 +24,11 @@ <h2>UI Settings</h2>
       <label for="choice-omit-emoji-in-display-names">Remove emoji from user display names</label>
     </div>
   </form>
-
+  
+  <h2>Themes
+  </h2>
+  <p>Changes to the theme (colours) of Pinafore can be made in <i>Settings &gt; Instances &gt; *your instance*</i>
+  </p>
```

Comment:
I like this change, but I'm not so sure about the second one. Thank you very much for this PR!

Reply:
As you see fit.

It's my pleasure.